examples/zephyr_example/src/main.c

```diff
@@ -18,7 +18,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <zephyr_port.h>
-#include <esp_loader.h>
+#include <esp_loader_io.h>
```

Comment:
What is the reasoning for this change?

Reply:
Nevermind, it looks like the port variants of functions have been used for resetting the target and changing the baudrate in the example. I will fix this so that it uses the public API and we don't need this change.